lib/builders/ProjectBuilder.js

```diff
@@ -342,15 +342,15 @@ class ProjectBuilder {
         const args = this.getArgs('clean', opts);
         return execa(wrapper, args, { stdio: 'inherit', cwd: path.resolve(this.root) })
             .then(() => {
-                fs.removeSync(path.join(this.root, 'out'));
+                fs.rmSync(path.join(this.root, 'out'));
```

Comment:
Does this need `{ recursive: true, force: true }`?

Reply:
It was not exactly clear if "out" was a directory or file. 

I thought this was an output log file.

I could not find in code where this directory or file is being created. But looking at the test spec, it refers to it as a directory. 

If correct, adding `{ recursive: true, force: true }` is required, which I just pushed.